synapse/lib/plex.py

```diff
@@ -27,8 +28,11 @@ def __init__(self):
 
         def fini():
             coro = self._onAsyncFini()
-            self.coroToSync(coro, timeout=2)
-            self.loop.stop()
+            try:
+                self.coroToSync(coro, timeout=.05)
+            except concurrent.futures.TimeoutError:
+                pass
```

Comment:
`_onAsyncFini` causes the ioloop to close, and this will cause the timeout to be thrown; so this is a exception to catch and ignore.

Reply:
This area will be redone anyway for async all the thingz, but this works for now